pyfatfs/PyFat.py

```diff
@@ -1096,7 +1096,10 @@ def mkfs(self, filename: str,
                 (4194304, 64)   # disks up to   2   GB, 32k cluster
             ],
             PyFat.FAT_TYPE_FAT12: [
-                (32768, 64)
+                (4084, 1),      # disks up to   2   MB, .5k cluster
+                (8168, 2),      # disks up to   4   MB,  1k cluster
+                (16336, 4),     # disks up to   8   MB,  2k cluster
+                (32672, 8)      # disks up to  16   MB,  4k cluster
```

Comment:
Awesome, thank you very much for your contribution as well as fixing the tests!

Do you think it would also make sense to add support for larger combinations such as:

- 65344/16: 32MB, 8k cluster
- 130688/32: 64MB/32k cluster
- 261376/64: 128MB/64k cluster

Maybe even the non-standard 522752/128 (256MB, 64k cluster)?

Reply:
Sure, I will add those. Originally I stopped the table at 4MB based on this comment from fatgen103.doc: "If your media is larger than 4 MB, do not bother with FAT12." But it should be fine to support the larger sizes.

Reply:
Thank you!